### PR TITLE
Refactor: migrate poll answer gathering

### PR DESCRIPTION
**Pull request by JoVictorNunes**
### What does this PR do?

Migrate poll answer gathering from Meteor to GraphQL.

### CODE REVIEW

**Comment by JoVictorNunes** [comment]
> polling container remains open after voting for an anonymous poll

@ramonlsouza Fixed by https://github.com/bigbluebutton/bigbluebutton/pull/19421/commits/b7fac7bfc048d58be86dbbacbbf82b59de494e19 and https://github.com/bigbluebutton/bigbluebutton/pull/19482/commits/1757c0b2f1ea11f20fc77bd4ac894d4e7e543d32.

File: bigbluebutton-html5/imports/ui/components/polling/polling-graphql/component.tsx [hunk 1]
```diff
@@ -0,0 +1,401 @@
+import React, {
+  ElementRef, useEffect, useMemo, useRef, useState,
+} from 'react';
+import { useMutation, useSubscription } from '@apollo/client';
+import { defineMessages, useIntl } from 'react-intl';
+import { Meteor } from 'meteor/meteor';
+import AudioService from '/imports/ui/components/audio/service';
+import Checkbox from '/imports/ui/components/common/checkbox/component';
+import PollService from '/imports/ui/components/poll/service';
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/polling/polling-graphql/component.tsx:9]: Prefer implements it on own service to isolate the component or move it on global scope

File: bigbluebutton-html5/imports/ui/components/polling/polling-graphql/component.tsx [hunk 2]
```diff
@@ -0,0 +1,401 @@
+import React, {
+  ElementRef, useEffect, useMemo, useRef, useState,
+} from 'react';
+import { useMutation, useSubscription } from '@apollo/client';
+import { defineMessages, useIntl } from 'react-intl';
+import { Meteor } from 'meteor/meteor';
+import AudioService from '/imports/ui/components/audio/service';
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/polling/polling-graphql/component.tsx:7]: move the AudioService to your service.ts, make it easier in future to remove.

File: bigbluebutton-html5/imports/ui/components/polling/polling-graphql/component.tsx [hunk 3]
```diff
@@ -0,0 +1,401 @@
...
+import Checkbox from '/imports/ui/components/common/checkbox/component';
+import PollService from '/imports/ui/components/poll/service';
+import useCurrentUser from '/imports/ui/core/hooks/useCurrentUser';
+import { isPollingEnabled } from '/imports/ui/services/features';
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/polling/polling-graphql/component.tsx:11]: Move it to service for the same reason of the AudioSettings

File: bigbluebutton-html5/imports/ui/components/polling/polling-graphql/component.tsx [hunk 4]
```diff
@@ -0,0 +1,401 @@
...
+  },
+});
+
+const validateInput = (i: string) => {
+  let input = i;
+  if (/^\s/.test(input)) input = '';
+  return input;
+};
+
+interface PollingGraphqlContainerProps {}
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/polling/polling-graphql/component.tsx:61]: Whay to have a empty declarion of an interface

**Review by Tainan404 — CHANGES_REQUESTED** [review]

**Review by Tainan404 — APPROVED** [review]
LGTM!

**Pull request merged**